openedx/features/edly/management/commands/tests/test_convert_oidc_clients_to_oauth2.py

```diff
@@ -0,0 +1,39 @@
+"""
+Tests the ``convert_oidc_clients_to_oauth2`` management command.
+"""
+
+import ddt
+from django.core.management import call_command
+from django.test import TestCase
+from oauth2_provider.models import get_application_model
+
+from openedx.core.djangoapps.oauth_dispatch.models import ApplicationAccess
+
+from ..convert_oidc_clients_to_oauth2 import Command
+from .factories import ClientFactory
+
+Application = get_application_model()
+
+
+@ddt.ddt
+class TestConvertOIDCClientsToOauth2(TestCase):
+    """
+    Tests the ``convert_oidc_clients_to_oauth2`` management command.
```

Comment:
nit: use double quotes ` Tests the "convert_oidc_clients_to_oauth2" management command.`

Reply:
Done